src/main/java/gameboy/gamepack/racinggame/data/entity/Referee.java

```diff
@@ -0,0 +1,35 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import java.util.*;
+
+import gameboy.gamepack.racinggame.data.vo.RaceLog;
+
+public class Referee {
+
+    private static final int MIN_POSITION_VALUE = 0;
+
+    private RaceRecorder recorder;
+
+    public Referee() {
+        this.recorder = new RaceRecorder();
+    }
+
+    public void record(List<Car> cars) {
+        recorder.record(cars);
+    }
+
+    public List<RaceLog> playback() {
+        return recorder.playback();
+    }
+
+    public List<String> getWinnerNames() {
+        RaceLog raceLog = recorder.playbackLastLog();
+        int winnerPosition = getWinnerPosition(raceLog);
+        return raceLog.getCars().stream().filter(car -> car.isWin(winnerPosition)).map(Car::getName).toList();
```

Comment:
```suggestion
        return raceLog.getCars()
        .stream()
        .filter(car -> car.isWin(winnerPosition))
        .map(Car::getName).toList();
```
체인이 여러개라면 가독성을 위해 끊어주시는 게 좋습니다.